.travis.yml

```diff
@@ -7,33 +7,41 @@ cache:
 
 env:
   global:
-    - RUN_PHPUNIT=1
     - RUN_PHPCS=0
 
 matrix:
   include:
-    - php: 7.0
-      env: RUN_PHPUNIT=0
-    - php: 7.0
-      env: RUN_PHPUNIT=0 COMPOSER_FLAGS="--prefer-stable --prefer-lowest"
-    - php: 7.1
-      env: RUN_PHPUNIT=0
-    - php: 7.2
-    - php: 7.3
-    - php: 7.4
-      env: RUN_PHPCS=1
+    - php: '7.2'
+      env: LARAVEL='5.5.*'
+    - php: '7.2'
+      env: LARAVEL='5.8.*'
+    - php: '7.2'
+      env: LARAVEL='^6.0'
+    - php: '7.3'
+      env: LARAVEL='5.5.*'
+    - php: '7.3'
+      env: LARAVEL='5.8.*'
+    - php: '7.3'
+      env: LARAVEL='^6.0'
+    - php: '7.4'
+      env: LARAVEL='5.5.*'
+    - php: '7.4'
+      env: LARAVEL='5.8.*'
+    - php: '7.4'
+      env: LARAVEL='^6.0' RUN_PHPCS=1
 
 before_script:
   - phpenv config-rm xdebug.ini || true
 
 install:
-  - |
-    if [[ $RUN_PHPUNIT = 0 ]]; then
-      # We assume the older PHP/Laravel versions and thus restore a working dependency system for them
-      composer remove --dev orchestra/testbench --no-interaction --no-update
-    fi
-  - travis_wait 20 travis_retry composer update ${COMPOSER_FLAGS} --no-interaction --prefer-dist
+  - composer require "illuminate/support:${LARAVEL}" --no-interaction --no-update
+  - composer require "illuminate/console:${LARAVEL}" --no-interaction --no-update
+  - composer require "illuminate/filesystem:${LARAVEL}" --no-interaction --no-update
+  - composer require --dev "illuminate/config:${LARAVEL}" --no-interaction --no-update
+  - composer require --dev "illuminate/view:${LARAVEL}" --no-interaction --no-update
+  - if [[ $LARAVEL = '5.5.*' ]]; then composer require --dev mockery/mockery --no-interaction --no-update; fi
+  - composer update --no-interaction --prefer-dist
 
 script:
+  - vendor/bin/phpunit
   - if [[ $RUN_PHPCS = 1 ]]; then vendor/bin/phpcs --standard=psr2 src/; fi
-  - if [[ $RUN_PHPUNIT = 1 ]]; then vendor/bin/phpunit; fi
```

Comment:
I decided to run phpunit tests first, i.e. make them more important to fail a build then the phpcs, hope that's good